core/src/main/scala/pureconfig/configurable/package.scala

```diff
@@ -74,4 +76,12 @@ package object configurable {
         keyFormatter(key) -> writerV.to(value)
       }.asJava)
     )
+
+  def baseJavaEnumReader[A <: java.lang.Enum[A]](
```

Comment:
Can you please name this `genericJavaEnumReader` for consistency with other readers here?